pyvista/themes.py

```diff
@@ -2090,6 +2094,7 @@ def __repr__(self):
             'Return Camera Position': 'return_cpos',
             'Hidden Line Removal': 'hidden_line_removal',
             'Anti-Aliasing': '_antialiasing',
+            'Split sharp edges': '_split_sharp_edges',
```

Comment:
Should we also add the feature angle to the repr?

Reply:
Done in c3d6f79d4e7b4cbea11f236ecbf324b7117cb371.